### ISSUE TITLE
Please update to Minecraft version 1.20 for your module.

### ISSUE THREAD

**Issue by surfishs** [UltimateBoomer/smoothboot-fabric#51]
Please update to Minecraft version 1.20 for your module.

**Comment by Kennybust** [comment]
> Please update to Minecraft version 1.20 for your module.

Agreed, it's still needs to be developed, because Mojang didn't fix a thing!

**Comment by safesaffawa** [comment]
我制作了一个 1.20 版本，它只更改了版本
https://github.com/safesaffawa/mc-smoothboot-1.20/releases/tag/1.19-1.7.1
